tests/zfs-tests/tests/functional/cli_root/zfs_send/zfs_send_007_pos.ksh

```diff
@@ -62,9 +62,9 @@ test_pool ()
 	log_must zfs snapshot $POOL/fs@a
 	while true; do
 		log_must find $mntpnt/ -type f -delete
-		sync
+		sync_pool
```

Comment:
```suggestion
		sync_pool $POOL
```

`sync_pool` assumes $TESTPOOL when not specified.  If we want to be able to use `sync_pool` as a drop in replacement for `sync` you'll want to change no arguments to mean "sync all pools". 

In general what I think we want is to establish a convention for how to use `sync_pool` as then do that consistently.

Reply:
There are a few cases where we need the force flag, without the pool parameter.
I am wondering if we want to make a new `sync_all_pools` (with optional force flag) instead.

Reply:
Sure, that works for me.  Plus it has the advantage that it's very clear what it does.